returnn/datasets/cached2.py

```diff
@@ -228,6 +228,15 @@ def get_target_list(self):
             keys.remove("data")
         return keys
 
+    def get_exact_complete_frac(self, sorted_seq_idx: int) -> float:
+        """
+        :return: fractional completion value for the given seq_idx
+        """
+        seq = self._get_seq(sorted_seq_idx)
```

Comment:
This is a bit costly. Doesn't it make more sense to first check `super().get_exact_complete_frac(sorted_seq_idx)` and if that throws `OptionalNotImplementedError`, then try this?

Reply:
Fine with me!

Reply:
With costly, I mean that `get_complete_frac` could trigger that some data would be loaded (via `_get_seq`) which was maybe not loaded yet and also there is no reason to load it, specifically not for just getting the complete_frac.

Reply:
I don't think `_get_seq` loads new segments unless you call `load_seqs`.

Reply:
Oh sorry, you are right, I remembered wrongly. So then it doesn't really matter.